go.mod

```diff
@@ -239,6 +239,7 @@ require (
 	github.com/grafana/grafana/pkg/aggregator v0.0.0-20250627191313-2f1a6ae1712b // @grafana/grafana-app-platform-squad
 	github.com/grafana/grafana/pkg/apimachinery v0.0.0-20250711114246-c9b2126c4ad5 // @grafana/grafana-app-platform-squad
 	github.com/grafana/grafana/pkg/apiserver v0.0.0-20250627191313-2f1a6ae1712b // @grafana/grafana-app-platform-squad
+	github.com/thomaspoignant/go-feature-flag v1.42.0 // @grafana/grafana-backend-group
```

Comment:
@hairyhenderson `v1.42.0` is the latest version that does not cause the dependency mismatch issue. I suggest either using it, or for the time being copying over the types we depend on from go-feature-flags to grafana.